third_party/lzma_sdk/download_lzma_sdk.py

```diff
@@ -14,6 +14,7 @@
 import shutil
 import subprocess
```

Comment:
This script should return immediately on non-linux host cause the lzma sdk is not needed for Android target so far.